translations/nl.json

```diff
@@ -0,0 +1,251 @@
+{
+  "nl": {
+    "INFO_TITLE": "Meon Informatie",
+    "INFO_DESC": "A multi-purpose bot capable of setting reminders, displaying news, detailed coronavirus stats, Hypixel stats, and more.",
+    "INFO_DEVS": "Ontwikkelaars",
+    "INFO_INV": "Uitnodiging",
+    "INFO_VOTE": "Stemmen",
+    "INFO_WEBSITE": "Website",
+    "INFO_DASH": "Dashboard",
+    "INFO_SUPPORT": "Hulp",
+    "INFO_LINKS": "Verwijzigingen",
+    "INFO_THANKS": "Speciale Dank",
+    "INFO_STATS": "Statistieken",
+    "INFO_SC": "Aantal Servers",
+    "INFO_MC": "Aantal Leden",
+    "INFO_CC": "Aantal Commands",
+    "INFO_TRANSLATORS": "Vertalers",
+    "PING": "Ping",
+    "UPTIME": "Tijd Online",
+    "INFO_TECH": "Technologie",
+    "INFO_LANG": "Taal",
+    "INFO_LIBRARY": "Library",
+    "INFO_HOST": "Gehost op GalaxyGate",
+    "INFO_SUPPORTED_LANGUAGES": "Ondersteunde Talen",
+    "PING_TEXT": "Bot Latentie",
+    "SYSTEM": "Systeem",
+    "BOT_SYSTEM_INFO": "Bot Systeem Informatie",
+    "BOT_SYSTEM_DESC": "Meer informatie is geven omdat het command is uitgevoerd door een Meon ontwikkelaar",
+    "CPU_USAGE": "CPU Gebruik",
+    "CPU_COUNT": "Logisch Aantal CPU's",
+    "MEMORY": "Geheugen",
+    "TOTAL_MEMORY": "Totaal Geheugen",
+    "AVAILABLE_MEMORY": "Beschikbaar Geheugen",
+    "MEMORY_USAGE": "Geheugengebruik",
+    "TOTAL_SPACE": "Totale Ruimte",
+    "AVAILABLE_SPACE": "Beschikbare Ruimte",
+    "DISK_USAGE": "Schijfgebruik",

```

Comment:
The [FREE_SPACE](https://github.com/BoupsInc/meon-translations/blob/master/translations/en.json#L37) key is missing